src/project/internal/projectconfiguration.cpp

```diff
@@ -118,6 +119,10 @@ void ProjectConfiguration::init()
 
     settings()->setDefaultValue(DISABLE_VERSION_CHECKING, Val(false));
 
+    settings()->setDefaultValue(DISABLE_BACKUP, Val(false));
+    settings()->setDescription(DISABLE_BACKUP, muse::qtrc("project", "Disable Backup").toStdString());
```

Comment:
For the UI, I would name it like this:
```suggestion
    settings()->setDescription(DISABLE_BACKUP, muse::trc("project", "Create backup of file on disk before saving new changes"));
```
(Also, you can use `trc` instead of `qtrc`; `trc` returns std::string directly)